packages/components/molecules/f-media-element/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-media-element",
   "description": "Fozzie Media Element - provides ability to display text and an image in different orientations",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
In changelog you added the changes as "To be added to next Release" so no need for version bump here

Reply:
fixed

Reply:
Its now telling me I need a version number

Reply:
I think the reason for the request for a version bump is the change to the tests. Happy to mark that Danger request as resolved and merge this if it's the only check that it's requesting.

Reply:
yea only change is to stories

Reply:
Cool – happy to bypass the danger checks once you've got two ✔️